### PR TITLE
Add isPlaylist to response JSON

### PR DESCRIPTION
**Pull request by devoxin**
This allows clients to identify Search or Playlist results and handle the response accordingly.

**This is a breaking change to clients as it modifies the JSON that is returned**

Partially fulfills #38

### CODE REVIEW

**Comment by PassTheMayo** [comment]
+1

File: LavalinkServer/src/main/java/lavalink/server/player/AudioLoader.java [hunk 1]
```diff
@@ -98,3 +103,13 @@ public void loadFailed(FriendlyException e) {
     }
 
 }
+
+class LoadResult {
+    public List<AudioTrack> tracks;
```
Comment (freyacodes) [inline LavalinkServer/src/main/java/lavalink/server/player/AudioLoader.java:113]: Why not make this immutable?

**Comment by dsgpaiva** [comment]
This is super useful, can someone be able to merge this PR?

**Comment by freyacodes** [comment]
It is planned for v3 because of its breaking changes. See the milestone

**Comment by dsgpaiva** [comment]
Ah, understood. Thanks

**Review by freyacodes — APPROVED** [review]

**Pull request merged**